app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_list: list[dict]) -> list:
+    person_list = []
+    for person in people_list:
+        person_obj = Person(person["name"], person["age"])
+        if "husband" in person:
+            if person["husband"] is not None:
+                person_obj.husband = person["husband"]
+        if "wife" in person:
+            if person["wife"] is not None:
+                person_obj.wife = person["wife"]
+        person_list.append(person_obj)
```

Comment:
can you create person_list with list comprehension? also, logic with creating wife and husband can be implemented in next loop

Reply:
sorry. my bad. my finish code was in other file.